libafl/src/events/llmp.rs

```diff
@@ -397,49 +433,34 @@ where
         };
         let count = events.len();
         events.drain(..).try_for_each(|(sender_id, event)| {
-            self.handle_in_client(state, sender_id, event, executor, scheduler)
+            self.handle_in_client(fuzzer, executor, state, sender_id, event)
```

Comment:
Why change the ordering? I don't care much, just asking because we had State as first param usually.

Reply:
I want to put fuzzer at first

Reply:
Leave state as second, maybe?